### PR TITLE
wayout: init at 1.1.3

### LINKED ISSUE #180946: Package request: wayout
**Project description**
A simple output management tool for wlroots based compositors implementing zwlr_output_power_management_v1

**Metadata**

* homepage URL: https://github.com/waycrate/wayout
* source URL: https://github.com/waycrate/wayout
* license: bsd
* platforms: linux

### PR DESCRIPTION
**Pull request by onny**
###### Description of changes

fixes https://github.com/NixOS/nixpkgs/issues/180946

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
Failing on Darwin. 
Set `meta.broken` accordingly

**Comment by onny** [comment]
@AndersonTorres Thank you for testing it. Disabled build for darwin

File: pkgs/tools/wayland/wayout/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  meta = with lib; {
+    description = "Simple output management tool for wlroots based compositors implementing";
+    homepage = "https://git.sr.ht/~shinyzenith/wayout";
+    license = licenses.bsd2;
+    maintainers = with maintainers; [ onny ];
+    broken = stdenv.isDarwin; # Build failed on Darwin
+  };
+
+}
```
Comment (SuperSandro2000) [inline pkgs/tools/wayland/wayout/default.nix:28]: NIT here shouldn't be a new line

File: pkgs/tools/wayland/wayout/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  };
+
+  cargoSha256 = "sha256-QlxXbfeWJdCythYRRLSpJbTzKkwrL4kmAfyL3tRt194=";
+
+  meta = with lib; {
+    description = "Simple output management tool for wlroots based compositors implementing";
+    homepage = "https://git.sr.ht/~shinyzenith/wayout";
+    license = licenses.bsd2;
+    maintainers = with maintainers; [ onny ];
+    broken = stdenv.isDarwin; # Build failed on Darwin
```
Comment (SuperSandro2000) [inline pkgs/tools/wayland/wayout/default.nix:25]: More important is how it failed. The comment should contain a short description of the error.

**Pull request merged**